pkg/runner/client.go

```diff
@@ -170,6 +170,13 @@ func CreateClient(r prometheus.Registerer, cfg *Config) (*pgclient.Client, error
 		if err != nil {
 			return nil, fmt.Errorf("error applying dataset configuration: %w", err)
 		}
+	} else {
+		// We apply downsampling settings even when DatasetConfig is not given, which is the most common case.
```

Comment:
Shouldn't we keep this feature off by default until it's battle tested? Otherwise if there is a bug it will affect all the users who upgrade

Reply:
I agree, let's keep it off by default.